deploy/cluster_role_binding.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: 3scale-operator
+subjects:
+  - kind: ServiceAccount
+    name: 3scale-operator
```

Comment:
I wonder what happens when `namespace` is not defined. Potentially, there could be one `3scale-operator` service account on each namespace.

Reply:
The yaml won't create if namespace is missed, now added back.